### PR TITLE
Feature/jawad feed

### PR DESCRIPTION
**Pull request by jnrizvi**

### CODE REVIEW

File: manager/frontend/src/containers/Feed.js [hunk 1]
```diff
@@ -5,10 +5,12 @@ import _ from 'lodash';
 import Navbar from '../components/Navbar/Navbar';
 import PostCreator from '../components/PostCreator/PostCreator';
 import PostSorter from '../components/PostSorter/PostSorter';
-import Posts from '../components/Posts/Posts';
+import Post from '../components/Posts/Post/Post';
```
Comment (kangxi11) [inline manager/frontend/src/containers/Feed.js:8]: why do you need to import this?

File: manager/frontend/src/containers/Feed.js [hunk 2]
```diff
@@ -54,7 +55,12 @@ export default function Feed() {
                     <div className={postClasses.join(' ')}>
                         <PostCreator />
                         <PostSorter />
-                        <Posts posts={temp_posts} />
+                        {simplifiedPosts.map( postData =>
```
Comment (kangxi11) [inline manager/frontend/src/containers/Feed.js:58]: shouldn't this be passed to Posts

**Review by kangxi11 — COMMENTED** [review]
just the thing about Post being imported into Feed instead of using Posts

File: manager/frontend/src/containers/Feed.js (hunk 1 above)
Comment (jnrizvi) [reply manager/frontend/src/containers/Feed.js:8]: So that an array of Post can be map'd in the Feed component

Comment (kangxi11) [reply manager/frontend/src/containers/Feed.js:8]: sure, we can refactor later if we need to

**Review by kangxi11 — APPROVED** [review]

File: manager/frontend/src/containers/Feed.js (hunk 2 above)
Comment (jnrizvi) [reply manager/frontend/src/containers/Feed.js:58]: postData is the post data for one post. simplfiedPosts is an array of postData

**Pull request merged**